src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Square extends Figure {
```

Comment:
The Square class should not have an empty line at the beginning of its declaration.